src/styles/card.css

```diff
@@ -0,0 +1,65 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
Set correct paddings to the main card instead of using left / right margins for all elements